docs-src/python/how-to-set-a-parent-close-policy-in-python.md

```diff
@@ -0,0 +1,28 @@
+---
+id: how-to-set-a-parent-close-policy-in-python
+title: How to set a Parent Close Policy in Python
+sidebar_label: Parent Close Policy
+description: Create an instance of the `ParentClosePolicy` class.
+tags:
+  - python
+  - developer-guide
+  - how-to
+---
+
+Set the `parent_close_policy` parameter inside the [`start_child_workflow`](https://python.temporal.io/temporalio.workflow.html#start_child_workflow) function or the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function to specify the behavior of the Child Workflow when the Parent Workflow closes.
+
+```python
+async def run(self, name: str) -> str:
```

Comment:
FWIW, I don't think we need an entire markdown file, w/ sentences and code snippet and everything, just for showing how to set one parameter.